### PR TITLE
feat: emit logs

### PR DESCRIPTION
**Pull request by sakulstra**

### CODE REVIEW

File: src/GovHelpers.sol [hunk 1]
```diff
@@ -182,13 +196,244 @@ library GovHelpers {
...
+   * @notice This enum contains all possible actions set states
+   */
+  enum ActionsSetState {
+    Queued,
+    Executed,
+    Canceled,
+    Expired
+  }
+
+  // Minimum allowed grace period, which reduces the risk of having an actions set expire due to network congestion
```
Comment (eboadom) [inline src/GovHelpers.sol]: Any reason to not get rid of this?

File: src/GovHelpers.sol [hunk 2]
```diff
@@ -182,13 +196,244 @@ library GovHelpers {
...
+        actionsSet.values[i],
+        actionsSet.signatures[i],
+        actionsSet.calldatas[i],
+        actionsSet.executionTime,
+        actionsSet.withDelegatecalls[i]
+      );
+      unchecked {
+        ++i;
+      }
+    }
```
Comment (eboadom) [inline src/GovHelpers.sol:312]: Better to keep the `ActionsSetExecuted` event?

Comment (sakulstra) [reply src/GovHelpers.sol:312]: idk, i removed all events as they seemed unncessary for me inside the mock, but can add back, no strong opinion

File: src/GovHelpers.sol [hunk 3]
```diff
@@ -182,13 +196,244 @@ library GovHelpers {
...
+
+    ActionsSet storage actionsSet = _actionsSets[actionsSetId];
+    actionsSet.targets = targets;
+    actionsSet.values = values;
+    actionsSet.signatures = signatures;
+    actionsSet.calldatas = calldatas;
+    actionsSet.withDelegatecalls = withDelegatecalls;
+    actionsSet.executionTime = executionTime;
+  }
```
Comment (eboadom) [inline src/GovHelpers.sol:391]: Same as before, no event `ActionsSetQueued` here?

**Review by eboadom — APPROVED** [review]

**Pull request merged**